ps_dataprivacy.php

```diff
@@ -56,14 +56,12 @@ public function install()
     {
         return (parent::install()
             && $this->registerHook('additionalCustomerFormFields')
-            && $this->registerHook('actionSubmitAccountBefore')
             && $this->installFixtures());
     }
 
     public function uninstall()
     {
         return ($this->unregisterHook('additionalCustomerFormFields')
-            && $this->unregisterHook('actionBeforeSubmitAccount')
```

Comment:
`actionSubmitAccountBefore` and `actionBeforeSubmitAccount` are actually the same hook (alias)